src/vm/win32threadpool.cpp

```diff
@@ -617,7 +617,7 @@ BOOL ThreadpoolMgr::SetMinThreads(DWORD MinWorkerThreads,
 
         if (GetForceMinWorkerThreadsValue() == 0)
         {
-            MinLimitTotalWorkerThreads = min(MinWorkerThreads, (DWORD)ThreadCounter::MaxPossibleCount);
+            MinLimitTotalWorkerThreads = max(1, min(MinWorkerThreads, (DWORD)ThreadCounter::MaxPossibleCount));
```

Comment:
I assume this is how the .NET Framework behaves?

Reply:
.NET Framework also allows 0 for the min thread count but would have the same issue. WorkerCounter's MaxWorking field is limited to this value on the low end and if it's allowed to become 0 through hill climbing thread adjustments, it triggers an assertion failure but also would require the starvation heuristic to kick in to pick up a work item. Marked for consideration for porting.